piety-promoted-v4/modernized/07-chapter-07.adoc

```diff
@@ -194,7 +204,10 @@ The following extracts from some of her letters, written near the close of life,
 very instructively describe the humility and liveliness
 of spirit which continued to prevail.
 
-1820+++.+++ Sixth month, 30. "`The future is entirely in obscurity to me,
+[.embedded-content-document.letter]
+--
+
+1820+++.+++ Sixth month, 30. The future is entirely in obscurity to me,
```

Comment:
hmmm... this chunk of letters doesn't have any long dash after the year. Maybe we should take them all out to make them the same? Or put them all in? I'll let you decide.... :)

Reply:
Sounds good...I think it looked a little better in the PDF view with the dashes...